### PR TITLE
Added pems_clearinghouse_start_date variable

### PR DESCRIPTION
**Pull request by kengodleskidot**
Variable added to dbt_project.yml and staging__station_raw model to minimize computation resources while we continue to develop/refine our downstream models. The variable is currently not added to station_meta and station_status based models due to their relatively small size and limited equipment updates by some districts (primarily district 11). This is associated with #181.

### CODE REVIEW

File: transform/dbt_project.yml
Comment (mmmiah) [inline transform/dbt_project.yml:1]: the logic seems good to me!

**Review by mmmiah — APPROVED** [review]
The applied logic make sense! Thanks!

**Review by britt-allen — APPROVED** [review]
This will be good to go in once CI is passing. If you make further edits while this is still a draft please re-request review. Thank you @kengodleskidot!

**Comment by kengodleskidot** [comment]
This PR closes #181.

**Comment by kengodleskidot** [comment]
> This will be good to go in once CI is passing. If you make further edits while this is still a draft please re-request review. Thank you @kengodleskidot!

@britt-allen we can discuss which other models may need to add this variable. My thought is that applying the variable to the staging_clearinghouse__station_raw model will limit the data being reprocessed for all associated downstream models but if you can think of any other models this would be good to implement with let me know.

**Comment by ian-r-rose** [comment]
This looks good to me, thanks @kengodleskidot! I agree that `stg_clearinghouse__station_raw` is the most important one to apply this to.

Since it's been in place for a week, have you noticed any improvements in development iteration times?

**Pull request merged**